### PR TITLE
[TD-4268] support continue query from last stop time . 

### PR DESCRIPTION
**Pull request by DuanKuanJun**
[TD-4268] Support start continue query stream from last stop time, not from begin.

### CODE REVIEW

File: src/client/src/tscStream.c [hunk 1]
```diff
@@ -572,6 +573,14 @@ static void tscCreateStream(void *param, TAOS_RES *res, int code) {
 
   pStream->stime = tscGetStreamStartTimestamp(pSql, pStream, pStream->stime);
 
+  // set stime with ltime if ltime > stime
+  const char* dstTable = pStream->dstTable? pStream->dstTable: "";
+  tscDebug(" CQ table=%s ltime is %"PRId64, dstTable, pStream->ltime);
+  if(pStream->ltime > 0 && pStream->ltime > pStream->stime) {
```
Comment (dapan1121) [inline src/client/src/tscStream.c:579]: 这里pStream->ltime不需要大于0吧，只要大于pStream->stime就行了吧

Comment (DuanKuanJun) [reply src/client/src/tscStream.c:579]: 修改了

**Comment by DuanKuanJun** [comment]
close

**Review by hzcheng — APPROVED** [review]

**Review by dapan1121 — APPROVED** [review]

**Comment by DuanKuanJun** [comment]
CLOSE THIS NEED OPEN NEW

**Pull request closed without merging**